src/components/EventSummary.js

```diff
@@ -57,24 +58,7 @@ const EventSummary = props => {
               </Grid>
             </Segment>
           </Grid.Column>
-          <Grid.Column width={4} className='event-info-videos'>
-            {event.videos
-              .filter(video => video.yt_video_id !== null)
-              .map(video => (
-                <Fragment key={video.id}>
-                  <Embed
-                    id={video.yt_video_id}
-                    placeholder={
-                      'https://img.youtube.com/vi/' +
-                        video.yt_video_id +
-                        '/3.jpg'
-                    }
-                    source='youtube'
-                  />
-                  <p>{video.title}</p>
-                </Fragment>
-              ))}
-          </Grid.Column>
+          <Videos name={event} id='event-info-videos' />
```

Comment:
@nataliyah123 What do you think of passing the videos as a prop? This way, the `Videos` component can stay dumb. And we can rename `id` to `className` to make it clear what it is.

```
<Videos videos={event.videos} className="event-info-videos" />
```
Thoughts?